tests/helpers.py

```diff
@@ -0,0 +1,176 @@
+from shared.django_apps.codecov_auth.models import BillingRate
+from shared.django_apps.codecov_auth.tests.factories import PlanFactory, TierFactory
+from shared.plan.constants import PlanName, PlanPrice, TierName
+
+
+def mock_all_plans_and_tiers():
```

Comment:
you are calling this explicitly in a ton of tests.

How is this being handled in production? Were these things created manually, or using a data migration?

Maybe it makes sense to introduce these kinds of "constants" using a data migration in the first place.
Otherwise, is it possible to auto-load these fixtures in all the tests that are using the django DB somehow?
Its a bit annoying that you have to call this manually in all those places. Better to just "have this data around" within your tests.